macaroons/constraints_test.go

```diff
@@ -132,6 +132,10 @@ func TestCustomConstraint(t *testing.T) {
 	require.False(t, macaroons.HasCustomCaveat(testMacaroon, "something"))
 	require.False(t, macaroons.HasCustomCaveat(nil, "foo"))
 
+	customCaveatCondition, err := macaroons.GetCustomCaveatCondition(testMacaroon, "unit-test")
```

Comment:
nit: line length

Reply:
Somehow missed these even after triple checking visually for line lengths.